doc/script_commands.txt

```diff
@@ -6133,6 +6133,58 @@ Examples:
 
 ---------------------------------------
 
+*pcblock <type>,<state>{,<account ID>};
+*checkpcblock {<account ID>};
+
+'pcblock' command prevents/allows the player from doing the given <type>
+of action according to the <state>.
+The <type> values are bit-masks, multiples <type> can be added to change the player action.
+
+The action is blocked when the <state> is true, while false
+allows the action again.
+
+'checkpcblock' command return the bit-mask value of the currently
+enabled block flags.
+
+The available <type> are:
+	PCBLOCK_MOVE
+	PCBLOCK_ATTACK
+	PCBLOCK_SKILL
+	PCBLOCK_USEITEM
+	PCBLOCK_CHAT
+	PCBLOCK_IMMUNE
```

Comment:
imo, rathena `*setunitdata` use `UMOB_DMGIMMUNE`
so I guess `PCBLOCK_DMGIMMUNE` might sound consistent with setunitdata counterpart

PS: I am posting once here because Atemo helped me fix PCBLOCK_EMOTION

Reply:
It wasn't out of curiosity? Thanks anyway